### PR TITLE
[8.0][procurement_mrp_no_confirm] Move from odoomrp/odoomrp-wip

### PR DESCRIPTION
**Pull request by andhit-r**

### CODE REVIEW

File: procurement_mrp_no_confirm/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+  .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Don't confirm MOs when procured
+==============
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/README.rst]: Put this of the same length of the title

File: procurement_mrp_no_confirm/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/OCA/manufacture
+2.  Add the path to this repository in your configuration (addons-path)
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
+5.  Search For *Procurement MRP no Confirm*
+6.  Install the module
+
+Configuration
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/README.rst]: Remove the empty section

File: procurement_mrp_no_confirm/README.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+======================
+
+* If the production order workflow has been altered in the transition
+  *prod_trans_draft_picking* to have another condition, when uninstalling this
+  module, you will need to restore the custom condition.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/README.rst]: Use new version of this section of https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: procurement_mrp_no_confirm/README.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/README.rst]: Please update my address to pedro.baeza@tecnativa.com

File: procurement_mrp_no_confirm/README.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+* Ainara Galdona <agaldona@avanzosc.es>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/README.rst:60]: Include yourself here

File: procurement_mrp_no_confirm/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+# © 2015 Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Procurement MRP no Confirm",
+    "version": "8.0.1.0.0",
+    "depends": ["mrp"],
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/__openerp__.py]: Please switch this line to my new company, Tecnativa

File: procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot]: Remove this file as is no longer needed.

File: procurement_mrp_no_confirm/__init__.py [hunk 8]
```diff
@@ -1,22 +1,9 @@
...
-#    along with this program.  If not, see http://www.gnu.org/licenses/.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# © 2015 AvanzOSC
+# © 2015 Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from . import models
+from . import models, tests
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/__init__.py]: tests folder is not imported. It's automatically used when testing server only.

File: procurement_mrp_no_confirm/tests/test_procurement.py [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    def test_1(self):
+        proc = self.env["procurement.order"].create(self.proc_data)
+
+        proc.run()
+        self.assertEqual(
+            proc.state,
+            "running")
+
+        # procurement should generate MO
+        self.assertIsNotNone(proc.production_id)
```
Comment (pedrobaeza) [inline procurement_mrp_no_confirm/tests/test_procurement.py]: This should be `self.assertTrue(proc.production_id)`, as the value is never None (maybe False, but not None).

File: procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot (hunk 7 above)
Comment (andhit-r) [inline procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot]: @pedrobaeza just to confirm, next time i move module from odoomrp i should remove .pot file?

Comment (pedrobaeza) [inline procurement_mrp_no_confirm/i18n/procurement_mrp_no_confirm.pot]: Yes, please, it's no longer needed as Transifex sync script generates it on each commit and can confuse users.

**Comment by pedrobaeza** [comment]
:+1: 

@oihane @anajuaristi please review this

**Comment by oihane** [comment]
LGTM :+1:

**Comment by andhit-r** [comment]
Hi @OCA/manufacturing-maintainers , would you kindly review this PR. Many thanks

**Comment by andhit-r** [comment]
Thanks @pedrobaeza

**Pull request merged**